src/xml/builder.cr

```diff
@@ -275,8 +275,17 @@ struct XML::Builder
     raise XML::Error.new("Error in #{msg}", 0) if ret < 0
   end
 
-  private def string_to_unsafe(string)
-    string ? string.to_unsafe : Pointer(UInt8).null
+  private def string_to_unsafe(string : String?)
```

Comment:
Just `def string_to_unsafe(string : Nil)` since you have the String overload below.

Reply:
I was going to comment exactly the same thing

Reply:
Fixed!